contracts/Nexus.sol

```diff
@@ -195,16 +194,15 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
 
     /// @notice Validates a signature according to ERC-1271 standards.
     /// @param hash The hash of the data being validated.
-    /// @param data Signature data that needs to be validated.
+    /// @param signature Signature data that needs to be validated.
     /// @return The status code of the signature validation (`0x1626ba7e` if valid).
     /// bytes4(keccak256("isValidSignature(bytes32,bytes)") = 0x1626ba7e
     /// @dev Delegates the validation to a validator module specified within the signature data.
-    function isValidSignature(bytes32 hash, bytes calldata data) external view virtual override returns (bytes4) {
+    function isValidSignature(bytes32 hash, bytes calldata signature) external view virtual override returns (bytes4) {
```

Comment:
why rename this signature over generic data?
typically signature is extracted way later

Reply:
just want it to be closer to 1271 spec